Packages/vcs/Lib/vcsvtk/fillareautils.py

```diff
@@ -33,8 +34,14 @@ def make_patterned_polydata(inputContours, fillareastyle=None,
     # Scaled the size to 2 times to make the pattern image of a finer resolution
     xBounds = bounds[1] - bounds[0]
     yBounds = bounds[3] - bounds[2]
-    xres = int(4.0*xBounds)
-    yres = int(4.0*yBounds)
+    if xBounds <= 1 and yBounds <= 1 and size is not None:
+        xBounds *= size[0]
```

Comment:
Sorry @chaosphere2112 that still does not sound right. I would not expect my view to change when taking a screenshot. Say, I set up the view which shows 20 pattern lines and take a screenshot that would resize the render window to half its size (say), then by this logic, I would get 10 pattern lines in the same area which is not a faithful reproduction of the scene. Am I right?